### PR TITLE
Shop

### PR DESCRIPTION
**Pull request by Hamza-Luciano**

### CODE REVIEW

**Review by FrancoisSalmon — APPROVED** [review]
ok for me

**Review by LuigiMenez — APPROVED** [review]
Ok for me.

File: frontend/src/pages/Shop/index.jsx [hunk 1]
```diff
@@ -67,18 +100,64 @@ export default function Shop() {
...
+                img2={product.img2}
+                img3={product.img3}
+                price={product.price}
+              />
+            );
+          })}
+        </article>
+        <article className="packPrice">
+          <Subtitle title="LE PACK ENTIER:" />
+          <Subtitle title="..€ au lieu de ..€" />
```
Comment (LoicBrassart) [inline frontend/src/pages/Shop/index.jsx:142]: ?

File: frontend/src/pages/Shop/index.jsx [hunk 2]
```diff
@@ -1,60 +1,93 @@
...
+      size5: "XL",
+    },
+    {
+      id: 2,
+      name: "name-2",
+      title1: "Sans flocage",
+      img1: "./src/assets/images/cards/products/jersey.png",
+      img2: "./src/assets/images/cards/products/jersey.png",
+      img3: "./src/assets/images/cards/products/jersey.png",
+      img4: "./src/assets/images/cards/products/jersey.png",
```
Comment (LoicBrassart) [inline frontend/src/pages/Shop/index.jsx:83]: ?

File: frontend/src/pages/Shop/index.jsx [hunk 3]
```diff
@@ -1,60 +1,93 @@
...
   ];
+  const jerseys = [
+    {
+      id: 1,
+      name: "name-1",
+      title1: "Avec flocage",
+      img1: "./src/assets/images/cards/products/jersey.png",
+      img2: "./src/assets/images/cards/products/jersey.png",
+      img3: "./src/assets/images/cards/products/jersey.png",
+      img4: "./src/assets/images/cards/products/jersey.png",
```
Comment (LoicBrassart) [inline frontend/src/pages/Shop/index.jsx:69]: ?

**Review by LoicBrassart — APPROVED** [review]
The code is okay, but:
- I asked you **not** to develop this page since we couldn't deliver this feature
- The feature is not really here, that's mainly a static page
- You used placeholders and galleries with the same picture

I don't get why you insist on it, but here you are

**Pull request merged**